temoa/extensions/modeling_to_generate_alternatives/hull.py

```
@@ -80,14 +96,15 @@ def update(self):
             self.good_points = self.cv_hull.points
             logger.info('Hull updated')
             self.volume = self.cv_hull.volume
-        except scipy.spatial._qhull.QhullError as e:
+        except Exception as e:
+            # scipy.spatial._qhull.QhullError may not be easily found by mypy
             logger.error(
                 'Attempt at hull construction from basis vectors failed.'
                 '\nMay be non-recoverable.  Possibly try a set of random vectors to initialize the '
                 'Hull.'
             )
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)